tests/suite/test_annotations.py

```diff
@@ -318,7 +322,7 @@ def test_upstream_zone_size_0(self, cli_arguments, kube_apis,
         if cli_arguments["ic-type"] == "nginx-plus-ingress":
             print("Run assertions for Nginx Plus case")
             assert "zone " in result_conf
-            assert " 256k;" in result_conf
+            assert " 512k;" in result_conf
```

Comment:
This is causing failures in the pipeline, although for some reason it is not showing up in the PR: https://gitlab.com/f5/nginx/kic/kubernetes-ingress/-/jobs/1623726013

Reply:
thanks for pointing out. forgot to update the template for N+ for Ingress, it still uses 256k default:
```
	zone {{$upstream.Name}} {{if ne $upstream.UpstreamZoneSize "0"}}{{$upstream.UpstreamZoneSize}}{{else}}256k{{end}};
```